### PR TITLE
DOC: add release note about revert for 1.0.5

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Whatsnew for https://github.com/pandas-dev/pandas/pull/34632

cc @simonjayhawkins @alimcmaster1

### CODE REVIEW

**Comment by alimcmaster1** [comment]
Reading from public buckets should work in 1.0.5 since we reverted right https://github.com/pandas-dev/pandas/pull/34632/files#diff-a37b395bed03f0404dec864a4529c97dR37 and thanks for doing this @jorisvandenbossche - I haven’t had a chance to!

**Comment by jorisvandenbossche** [comment]
@alimcmaster1 yes, that's what the whatsnew is supposed to say. But my wording might be confusing?

**Comment by alimcmaster1** [comment]
> @alimcmaster1 yes, that's what the whatsnew is supposed to say. But my wording might be confusing? 

Apols I misread - makes sense!

File: doc/source/whatsnew/v1.0.5.rst [hunk 1]
```diff
@@ -15,16 +15,18 @@ including other versions of pandas.
 
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
--
--
+
+- Fix regression in :meth:`read_parquet` when reading from file-like objects or
+  from public S3 buckets. Note this disables the ability to read from
```
Comment (simonjayhawkins) [inline doc/source/whatsnew/v1.0.5.rst:20]: perhaps ref issue #26388 here directly

**Review by simonjayhawkins — APPROVED** [review]
Thanks @jorisvandenbossche lgtm

**Comment by jorisvandenbossche** [comment]
@MeeseeksDev backport to 1.0.x

**Pull request merged**